clang/lib/Analysis/UnsafeBufferUsage.cpp

```diff
@@ -959,12 +966,12 @@ class UnsafeBufferUsageAttrGadget : public WarningGadget {
 /// perform buffer operations that depend on the correctness of the parameters.
 class UnsafeBufferUsageCtorAttrGadget : public WarningGadget {
   constexpr static const char *const OpTag = "cxx_construct_expr";
-  const CXXConstructExpr *Op;
+  const Expr *Op;
```

Comment:
Is this change still necessary? I think it was included by accident.

Reply:
Yes. That was accidentally edited. Let me revert this.